content/telegraf/v1.27/data_formats/input/_index.md

```diff
@@ -35,4 +37,9 @@ desired parser:
   data_format = "json_v2"
 ```
 
+## Input parser plugins
+
+When you specify a `data_format` in an [input plugin](/{{< latest "telegraf" >}}/plugins/inputs/) configuration that supports it, the input plugin uses the associated [parser plugin](https://github.com/influxdata/telegraf/tree/master/plugins/parsers) to convert data from its source format into Telegraf metrics.
```

Comment:
> Looks like parsers are something we need to add to the public-facing docs.

That was my plan, initially. However, we're actually using Data Formats as an alias for them - except for the XPath parser, parser:format is 1:1. And `data_format` is what the user interacts with... it didn't seem like drawing the distinction would add much value and might be more confusing. For Telegraf devs, though, it would be useful.